src/shared/components/Pagination/Pagination.style.tsx

```diff
@@ -47,8 +48,8 @@ export const ThreeDotsWrapper = styled.div`
 `
 
 const buttonActiveState = css`
-  background-color: ${colors.transparentPrimary[12]};
-  color: ${colors.gray[50]};
+  background-color: ${cVar('colorCoreNeutral800Lighten')};
+  color: ${cVar('colorCoreNeutral50')};
 `
 
 export const PaginationButton = styled.button<PaginationButtonProps>`
```

Comment:
```suggestion
export const PaginationButton = styled(Button)<PaginationButtonProps>`
```

Reply:
hmm, actually we could just simplify this to: 
```
export const PaginationButton = styled(Button)<PaginationButtonProps>`
  ${square(PAGINATION_BUTTON_WIDTH)};

  color: ${colors.gray[300]};
  border-radius: 100%;
  user-select: none;

  :hover {
    color: ${colors.gray[50]};
  }

  ${({ isActive }) => isActive && buttonActiveState};
`
```
and in the component we should just add variant `tertiary`

```
      <PaginationButton
            variant="tertiary"
            isActive={internalPage ? internalPage === pageItem : pageItem === 1}
            key={idx}
            onClick={() => handleChangePage(pageItem)}
          >
```